packages/flutter_tools/test/integration.shard/test_utils.dart

```diff
@@ -81,16 +81,20 @@ Future<void> pollForServiceExtensionValue<T>({
   @required FlutterTestDriver testDriver,
   @required String extension,
   @required T continuePollingValue,
-  @required T expectedValue,
+  @required Matcher matches,
   String valueKey = 'value',
 }) async {
-  for (int i = 10; i < 10; i++) {
+  for (int i = 0; i < 10; i++) {
```

Comment:
This might need more time on the bots

Reply:
Are you suggesting to increase the test timeout or to loop more times?

Reply:
either works